zproject/settings.py

```diff
@@ -159,6 +159,7 @@ def get_secret(key):
                     'REMOTE_POSTGRES_SSLMODE': '',
                     'GOOGLE_CLIENT_ID': '',
                     'DBX_APNS_CERT_FILE': None,
+                    'ENABLE_FILE_LINKS' : False,
```

Comment:
You should also add a patch to `zproject/local_settings_template.py` documenting the feature and providing a clear place to set this new setting.